README.md

```diff
@@ -1,8 +1,72 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Harikrishna S Pillai . Jarvis Consulting
 
+A Software engineering enthusiast who is passionate in developing applications. Graduated in 2020 from Centennial College with focus on embedded systems where I developed several programs written in assembly language which communicated with microcontrollers. Working at Jarvis helped me realize my passion for programming and developing real world applications. Looking to leverage my acumen by working on challenging products and services.
+
+## Skills
+
+**Proficient:** Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, Docker, Java
+
+**Competent:** Python, Tableau, MS Excel, Dart, Objective C
+
+**Familiar:** C++, JavaScript, CSS, HTML
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai)
+
+
+**Linux Cluster Monitoring Agent** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/linux_sql)]: The monitoring agent does the job of actively keeping an eye on the usage statistics which helps in effective resource management.This is possible with help of several different technologies working in tandem. The monitoring agent works on each node inserting values every minute from the node into postgres SQL database which mounted on a container using Docker.
```

Comment:
The monitoring agent does the job of actively keeping an eye on the usage statistics which helps in effective resource management. This is possible with help of several different technologies working in tandem. The monitoring agent works on each node inserting values every minute from the node into postgres SQL database which mounted on a container using Docker.

lease include the following keywords: CentOS, Bash/Shell, docker, RDBMS/SQL/Postgresql, crontab, data analytics/SQL


Developed XYZ for purposes using A, B, C.

Reply:
Included all the mentioned keywords